docs/Why-Mock.md

```diff
@@ -0,0 +1,107 @@
+# Why Mock?
+
+This document tries to answer two questions:
+
+1. Why you should use Mock instead of lower-level tooling like `rpmbuild`
+2. Why you should not try to reimplement Mock and rather join the project
+
+
+## Mock builds packages inside an isolated environment
+
+Building RPM packages is inherently dangerous because specfiles are written in a
+Turing-complete language. Building an untrusted package can easily lead to a
+compromised system. This is also true for self-written packages. One small bug
+like `rpm -rf %{?nonexisting}/` can wipe your entire system if you are unwise
+enough to run `rpmbuild` as `root`. Or at least your user home directory.
+
+Mock builds packages inside of `systemd-nspawn` containers to prevent such
+disasters from happening.
+
+
+## Mock can build packages for different distributions
+
+Mock comes with the `mock-core-configs` package which provides a list
+[configuration files for many RPM-based distributions][mock-core-configs].
+Mock allows building a package for any of them, regardless of what distribution
+is installed on your host system.
```

Comment:
Can you emphasize how mock is doing that, e.g. that on Fedora 42 machine we install Mageia build chroot, and we build "inside" that chroot?  This is probably the #1 reason.

This is probably the #1 reason to use Mock.